README.md

```diff
@@ -211,6 +208,8 @@ If you like this project follow me on Github and give it a star to motivate buil
 
 I would like to thank the Microverse Community, My stand up team and coding partners who we work tirelessly to make it possible.
 
+Lots of love and appreciation to [Cindy Shin] (https://www.behance.net/adagio07) the [author of the original design] (https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), in Creative Commons platform.
```

Comment:
- [OPTIONAL] Well done for giving the appropriate credits to Cindy Shin. What do you think of removing the space between these brackets 👉 `[ ]( )` ✨